util/README.md

```diff
@@ -0,0 +1,42 @@
+# Utility
+
+This directory has the following:
+
+* `/container` - Has the docker file used for this repo.
+* `/scripts` - Contains useful scripts. Right now we have a template generation tool.
+* `/cfg` - Has configuration files for templates used for the template generation.
+* `/templates` - Contains several templates for different files.
+
+# Docker Usage
+
+You can build the docker with:
+
+```bash
+docker build -t ghcr.io/kuleuven-micas/snax-cocotb .
```

Comment:
isn't it `docker build util/container -t snax-cocotb`?

Reply:
Good point haha too much copy-pasting hahaha

Reply:
Fixed!